src/objset.c

```diff
@@ -595,14 +616,16 @@ void ClearObjMap(Obj map) {
  *  This function returns all values from the map.
  */
 
-Obj ObjMapValues(Obj set) {
-  UInt len = CONST_ADDR_WORD(set)[OBJSET_USED];
-  UInt size = CONST_ADDR_WORD(set)[OBJSET_SIZE];
+Obj ObjMapValues(Obj (map))
```

Comment:
some weird stuff is going on here, and two lines lower, with double brackets.

Reply:
Oops, will fix